src/pybind/CMakeLists.txt

```diff
@@ -1,33 +1,51 @@
-find_package(Cython REQUIRED)
 include(Distutils)
 
 set(CYTHON_MODULE_DIR ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/cython_modules)
 
-add_subdirectory(rados)
-add_subdirectory(rbd)
-add_subdirectory(cephfs)
+foreach(python_version 3 2)  # Keep the default version last
+  if(${python_version} EQUAL 2)
+    set(PYTHON_VERSION "")
+  else(${python_version} EQUAL 2)
+    set(PYTHON_VERSION ${python_version})
+  endif(${python_version} EQUAL 2)
 
-add_custom_target(cython_modules ALL
-  DEPENDS cython_rados cython_cephfs cython_rbd)
+  find_package(Python${PYTHON_VERSION}Interp ${python_version} QUIET)
```

Comment:
Just copied what was there before - no `REQUIRED` but a custom error message right after. It might make it clearer what version of Python is needed.
If you think that needs to be changed, please reply again.